### ISSUE TITLE
When we press enter for input elements, disable focus

### ISSUE THREAD

**Issue by devcem** [playcanvas/editor#1008]
That was the face before the update, but for some reason this doesn't work. Please bring this feature back. I always find myself in a situation where I have some number and f in input.

**Comment by willeastcott** [comment]
Sorry, can you clarify the problem here please? You're saying there's a regression in PCUI?

**Comment by slimbuck** [comment]
Yes I've noticed this too. I was able to set scale x, y, z easily in the editor by selecting x then tabbing to y and z. After setting z I could press enter and continue working.

That doesn't work anymore. Pressing enter then f results in scale.z being reset to 0.

**Comment by willeastcott** [comment]
Really sorry about this @devcem - I only just got around to investigating this and I believe I've submitted the appropriate fix: https://github.com/playcanvas/pcui/pull/316

Assuming that's approved soon, I'll merge it and get it deployed for you.
